### DISCUSSION TITLE
Purpose of a membership group

### DISCUSSION THREAD

**Discussion by fostermi** [drakkan/sftpgo#1634] [Q&A]
I'm looking at how to map LDAP groups to groups in sftpgo and thought that perhaps membership groups would be the way to go, with each LDAP group having an specific bucket on S3. However, doing some testing, and then reading the documentation, it seems that virtual folders for a group are not inherited when the group is assigned to a user as a membership group. The documentation says
```
The following settings are inherited from the primary and secondary groups:

    virtual folders, file patterns, permissions: they are added to the user configuration
```
The only way a user can view folders assigned to a group is if its a Primary or Secondary group. So, if that's the case, what exactly does a membership group do for a user if the user inherits no folders or storage from that group configuration?

**Comment by drakkan** [comment]
It is useful to restrict rules in the EventManager for example
